### DISCUSSION TITLE
Traceability matrix report

### DISCUSSION THREAD

**Discussion by mariaklimenko** [allure-framework/allure2#1874] [TestOps]
Hello! We are considering to onboard Allure TestOps in our project. One of the requirements is to be able to automatically generate a traceability matrix report and a test coverage report (we keep requirements in Jira Cloud). Can you please confirm that it is possible to do this in Allure TestOps? And if so then, please, point me to the corresponding doc. Thanks in advance!

**Comment by cheshi-mantu** [comment]
Hi @mariaklimenko 
could you please provide an example or describe in details what and you want to see?
